main.tf

```diff
@@ -1,7 +1,6 @@
 locals {
   cors_rule = var.cors_rule != null ? { create = true } : {}
 }
-
```

Comment:
Can this empty line be added back please

Reply:
Removed that by mistake, will add back